frontend/src/Editor/Components/Container.jsx

```diff
@@ -12,7 +12,7 @@ export const Container = function Container({
   styles,
   darkMode,
 }) {
-  const { visibility, disabledState, borderRadius,borderColor } = styles;
+  const { visibility, disabledState, borderRadius, borderColor } = styles;
```

Comment:
the pipeline was failing because of a lint issue here after pulling the master branch. So had to make this change.